### PR TITLE
Add Serializable type into the docs

### PR DESCRIPTION
**Pull request by JoelEinbinder**
This patch converts all of our spread operators to be documented as <...Value>. It adds a Serializable type with links to the documentation on JSON.stringify.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -390,7 +390,7 @@ The function is invoked after the document was created but before any of its scr
 #### page.exposeFunction(name, puppeteerFunction)
 - `name` <[string]> Name of the function on the window object
 - `puppeteerFunction` <[function]> Callback function which will be called in Puppeteer's context.
-- returns: <[Promise]> Promise which resolves with the result of `puppeteerFunction`.
```
Comment (JoelEinbinder) [inline docs/api.md:392]: This promise did not contain the result of `puppeteerFunction`

File: docs/api.md [hunk 2]
```diff
@@ -1242,3 +1241,4 @@ Contains the URL of the response.
 [Tracing]: #class-tracing "Tracing"
 [ElementHandle]: #class-element "ElementHandle"
 [UIEvent.detail]: https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail "UIEvent.detail"
+[Serializable]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify "Serializable"
```
Comment (paulirish) [inline docs/api.md]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Description ?

File: docs/api.md (hunk 1 above)
Comment (aslushnikov) [reply docs/api.md:392]: it does: https://github.com/GoogleChrome/puppeteer/blob/master/test/test.js#L761

**Review by aslushnikov — COMMENTED** [review]
Thanks Joel!

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**